practice/pig-latin.py

```diff
@@ -1,58 +1,122 @@
 import string
 
-"""
-Translates english into pig-latin
+""" Translates english into pig-latin.
 Note: I still have not figured out to solve for punctuation that begins a word as of yet.
-Future Goal #1: Solve for Capital and Punctuation on one word [e.g. No.]
-Future Goal #2: Solve for apostraphes
-Future Goal #3: Solve for punctuation that begins a word, [e.g. (Sam) "Bob" or 'Tiffany' 'Tis]
-Future Goal #4: Solve for capitals in the middle of a word, [e.g. McDonalds]
-Future Goal #5: Solve for multiple punctuation [e.g. "Is that butter?" or (Don't turn off the lights.)]
+Future Goal #1: Solve for punctuation that begins a word, [e.g. (Sam) "Bob" or 'Tiffany' 'Tis]
+Future Goal #2: Solve for multiple punctuation [e.g. "Is that butter?" or (Don't turn off the lights.)]
 Note: With these goals, I'd like to avoid ruining punctuation around short words such as I and My.
+EDIT 12-July-2016: Altering code to be a series of functions, solved for some apostrophes and capitals.
+EDIT 12-July-2016: Added list of words in English that start with Y-vowel for higher accuracy.
 """
 
-# 1. Setup
-FIRST_VOWEL = 'a e i o u'.split()
-LATER_VOWEL = 'a e i o u y'.split()
+
+# 1. Setup Define/Functions
+
+
+FIRST_VOWEL = 'a e i o u A E I O U'.split()
+LATER_VOWEL = 'a e i o u y A E I O U Y'.split()
+SPECIAL_CASE = 'Ybe ybe Ycleped ycleped Yclept yclept Ysgard Ydo ydo Ydrad ydrad Yfere yfere Ygdrasyl ygdrasyl Yghe'\
+               'ygh Ygo ygo Yground yground Ypres ypres Ypsiliform ypsiliform Ypsiloid ypsiloid Yren yren Ysame ysame '\
+               'Ytterbic ytterbic Ytterbium ytterbium Yttria yttria Yttric yttric Yttriferous yttriferous Yttrious ' \
+               'yttrious Yttrium yttrium Yttro-cerite yttro-cerite Yttro-columbite yttro-columbite Yttro-tantalite ' \
+               'yttro-tantalite Yvel yvel Ywar ywar Ywis ywis'.split()
 PUNCTUATION_LIST = list(string.punctuation)
 CAPITALS_LIST = list(string.ascii_uppercase)
-new_punctuation = ''
-new_phrase = ''
 
-# 2. Input
 
-english_phrase = input("What would you like to Anslatetray? ").split()
+# 2. Define Functions
+
+
+def get_phrase_casted_into_list():
+    """Aquires english phrase from user input and casts it as a variable"""
+    typed_phrase = input("What would you like to Anslatetray? ").split()
+    return typed_phrase
+
+
+def pull_english_word_from_list (phrase, i):
```

Comment:
I think this falls over the line of being a function that is extraneous. The subscript operator is only used for picking a specific item out of a list, and so it wouldn't be mis-interpreted as anything else and I think giving it a name is extraneous. That's okay! Honing in on the balance between too many functions and too few functions is hard.